model/version.go

```diff
@@ -106,6 +107,27 @@ func (v *Version) AddSatisfiedTrigger(definitionID string) error {
 	return errors.Wrap(AddSatisfiedTrigger(v.Id, definitionID), "error adding satisfied trigger")
 }
 
+func (v *Version) ChangeStatus(newStatus string) error {
```

Comment:
Can we call this UpdateStatus to match the build version of this function / it's more consistent with what the function is doing

Reply:
Also, this function may be better suited for version_db.go

Reply:
changed. I left it here because only part of it deals with db changes to versions